tagging/utils.py

```diff
@@ -3,12 +3,25 @@
 calculation.
 """
 import math
-import types
 
 from django.db.models.query import QuerySet
-from django.utils.encoding import force_unicode
+try:
+    from django.utils.encoding import force_unicode
+except ImportError:
+    def force_unicode(s, *args, **kwargs):
+        return s
```

Comment:
I'm not too familiar with crosscompatibility issues,  but isn't https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.encoding.force_text the new name?  (the note below on force_unicode would make me think so)